crates/api/src/builder/api.rs

```diff
@@ -282,21 +275,10 @@ where
             .find(|duty| duty.slot == slot)
             .ok_or(BuilderApiError::ProposerDutyNotFound)?;
 
-        debug!(slot, duty = ?duty, "delegations -- found proposer duty");
-
         let pubkey = duty.entry.message.public_key.clone();
+        let delegations = Json(api.auctioneer.get_validator_delegations(pubkey).await?);
```

Comment:
why did you remove the logs here?

Reply:
There are already a ton of logs in Helix and I wanted to remove everything that's non-essential even at the debug level.
This log doesn't give anything helpful, if a proposer duty is found then we simply don't see the error which is enough for understanding if the code is working correctly.

Reply:
Gotcha, ty